modules/acquia_cms_headless/src/Plugin/AcquiaCmsHeadless/HeadlessApiUsers.php

```diff
@@ -74,12 +74,20 @@ class HeadlessApiUsers extends AcquiaCmsDashboardBase {
    */
   protected $module = 'consumers';
 
+  /**
+   * Provides headless role label.
+   *
+   * @var string
+   */
+  protected $headlessRoleLabel;
+
   /**
    * {@inheritdoc}
    */
   public function __construct(StateInterface $state, ModuleHandlerInterface $module_handler, LinkGeneratorInterface $link_generator, InfoParserInterface $info_parser, EntityTypeManagerInterface $entity_type_manager, StarterkitNextjsService $starterkit_nextjs_service) {
     parent::__construct($state, $module_handler, $link_generator, $info_parser);
     $this->entityTypeManager = $entity_type_manager;
+    $this->headlessRoleLabel = $entity_type_manager->getStorage("user_role")->load('headless')->label();
```

Comment:
@deepakmishra2 user_role should be in single quotes

Reply:
Done